ubuntu-installer-classic-2304-amd64-dangerous.json

```diff
@@ -0,0 +1,34 @@
+{
+    "type": "model",
+    "series": "16",
+    "authority-id": "canonical",
+    "brand-id": "canonical",
+    "model": "ubuntu-installer-classic-2304-amd64-dangerous",
+    "architecture": "amd64",
+    "timestamp": "2023-06-08T19+00:00.0Z",
+    "base": "core22",
+    "grade": "signed",
```

Comment:
hm, is this supposed to be `signed` here as well, or `dangerous` since it's a dangerous model?